projects/health-warden.json

```diff
@@ -15,48 +15,43 @@
   "desc_de": "HealthWarden ist eine mobile und Desktop-App für die komplexe ambulante Versorgung. Die HealthWarden-App verkürzt die Krankenhausaufenthaltszeit und sorgt für einen reibungslosen und sicheren Kontakt zwischen Arzt und Patient während der ambulanten Behandlung. Es führt regelmäßige Zustands- und Gesundheitschecks des Patienten mit modernsten prädiktiven KI-Algorithmen durch. Alle potenziellen Probleme werden dem Arzt mithilfe von Anomalieerkennungstools gemeldet. Jeder Patient erhält eine strukturierte personalisierte Versorgung basierend auf den individuellen Anforderungen, während Ärzte gut strukturierte und leicht zugängliche Daten zu jedem Patienten sowie seiner Kohorte erhalten.",
   "desc_short_cs": "Mobilní a desktopová aplikace pro komplexní ambulantní péči. Pacienti používají mobilní aplikaci k zaznamenávání příznaků mimo nemocniční prostředí a lékaři používají webovou aplikaci ke sledování pokroku svých pacientů a získávání objektivních dat.",
   "desc_short_en": "A mobile and desktop app for complex outpatient care. Patients use mobile app to record their symptoms outside hospital settings and doctors use the web app to monitor the progress of their patients and receive objective data.",
-  "contact_note_cs": "Máte zájem o využití aplikace HealthWarden ve Vaší nemocnici? Kontaktujte nás e-mailem a přípravíme Vám nabídku na míru.",
-  "contact_note_en": "Would you like to use HealthWarden app in your hospital? Contact us by e-mail below a to get a customized offer.",
+  "contact_note_cs": "Zaujaly Vás možnosti Healthwardenu a chcete zjistit, jak by mohl fungovat ve Vaší nemocnici? Kontaktujte nás e-mailem pro nabídku na míru.",
+  "contact_note_en": "Are you interested in the possibilities of Healthwarden and want to find out how it could work in your hospital? Contact us by email for a tailor-made offer.",
```

Comment:
```suggestion
  "contact_note_en": "Are you interested in the possibilities of HealthWarden and want to find out how it could work in your hospital? Contact us by email for a tailor-made offer.",
```